bitcoin/src/crypto/key.rs

```diff
@@ -201,7 +198,7 @@ impl PublicKey {
         secp: &Secp256k1<C>,
         msg: &secp256k1::Message,
         sig: &ecdsa::Signature,
-    ) -> Result<(), Error> {
+    ) -> Result<(), secp256k1::Error> {
         Ok(secp.verify_ecdsa(msg, &sig.signature, &self.inner)?)
```

Comment:
In 4717cb9a384afc21817a1fba35573311e4ce2b0b:

You can drop the `Ok()` and the `?` now.

Reply:
Ah thanks!